cves/kernel/CVE-2021-27365.yml

```diff
@@ -75,7 +79,7 @@ bugs_instructions: |
     * Mentioned in mailing list discussions
     * References from NVD entry
     * Various other places
-bugs: []
+bugs: [1182715,]
```

Comment:
you have an extra "," that you don't need       :)

Reply:
Thanks!